### DISCUSSION TITLE
Can I set the heat source at the boundary?

### DISCUSSION THREAD

**Discussion by seongwonHam** [idaholab/moose#22713] [Q&A Modules: General] [answered]
Hello, guys.

Now I want to simulate the PC radiator (for cooling) using MOOSE, but I have some trouble setting the boundary condition.
To confirm the temperature change on the hot side, I couldn't set the temp BCs at the hot side (bottom). 
(because If I set the boundary condition, the temperature doesn't change at the simulation!)

So, I want to set the heat source at this simulation, but I don't know how can I make the heat source only on the bottom hot side.
When I used the "Heat Source" at the kernel, all of the mesh make the volumetric heat, and my plan would fail. 

Please let me know how I can set the heat source at the boundary.
Thanks a lot for your kindness!

- Ham

**Comment by maxnezdyur** [comment, marked as answer]
I think you may be looking for ConvectiveHeatFluxBC or a Neumann BC. 
https://mooseframework.inl.gov/source/bcs/ConvectiveHeatFluxBC.html

**Reply by GiudGiud** [reply]
Agreed. The boundary conditions will let you specify a heat flux, either from a heat source or towards the heat sink.
